conjure-java-jaxrs-client/src/test/java/com/palantir/conjure/java/client/jaxrs/JaxRsClientConfigRefreshTest.java

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 
-package com.palantir.remoting3.jaxrs;
+package com.palantir.conjure.java.client.jaxrs;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import com.palantir.remoting3.clients.ClientConfiguration;
-import com.palantir.remoting3.ext.refresh.Refreshable;
+import com.palantir.conjure.java.clients.ClientConfiguration;
+import com.palantir.conjure.java.ext.refresh.Refreshable;
```

Comment:
are we fine with `com.palantir.conjure.java.ext`? just want to call that out

Reply:
I think the more trivial the change is, the better for migration (in most places except client/servers, we're just replacing the package prefix).
Don't see a compelling reason to name this something else.